scripts/bazel-bisect.sh

```diff
@@ -79,7 +79,8 @@ cd "$WORKING_DIR"
 # Call the newly built Bazel. If it fails, save the exit code but don't stop
 # the script.
 result=0
-"$TMP_BIN" $BAZEL_ARGUMENTS || result=$\? && true
+"$TMP_BIN" clean || true
```

Comment:
Let's key this on a BAZEL_BISECT_WITH_CLEAN=1?

Reply:
Done, except BAZEL_BISECT_WITH_CLEAN=y because integer booleans are wrong. :)